### DISCUSSION TITLE
Allow desktop app to have multiple windows

### DISCUSSION THREAD

**Discussion by gigawhitlocks** [mattermost/community#17] [Enhancements]
It would be nice to be able to have multiple channels, servers, or teams, open simultaneously without creating new browser windows, by having the Mattermost Desktop App be capable of spawning multiple windows.

This would also fit with the paradigm of application switching that is dominant on the 3 desktop OSes better than having multiple browser windows to achieve the same behavior.

**Comment by matthewbirtch** [comment]
Yes! This would be awesome to be able to pop out a particular conversation or channels as well

**Comment by esethna** [comment]
Upvoted this because I think it would be awesome, but we definitely need to be careful about letting the user get into a state where they have multiple window of mattermost that are all getting notifications and are interchangable would get messy etc. Pop-out chats and limitations on what we can and can't do within a pop-out will be important! But I agree it needs further exploration!

**Comment by Willyfrog** [comment]
Maybe this is something we can explore with the suite dropdown? Have a popout button from the main window

**Reply by esethna** [reply]
Yes totally
